x/group/keeper/keeper.go

```diff
@@ -416,6 +416,8 @@ func (k Keeper) TallyProposalsAtVPEnd(ctx sdk.Context) error {
 				return sdkerrors.Wrap(err, "proposal update")
 			}
 		}
+		// Note: We do nothing if the proposal has been marked as ACCEPTED or
```

Comment:
How about just logging so that in the future we can know if it was accepted or rejected given this can easily slip the mind since we aren't doing anything?

Reply:
By logging you mean a `ctx.Logger().Log`? Isn't it too much to show one additional line in the CLI on each block?